test-app/.prettierrc.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
 module.exports = {
+  plugins: ['prettier-plugin-ember-template-tag'],
+  singleQuote: true,
+  templateSingleQuote: false,
   overrides: [
     {
-      files: '*.{js,ts}',
+      files: '*.gjs',
```

Comment:
is this not automatic? 🤔 

maybe I missed something the the README for this plugin

Reply:
Yeah, probably this shouldn't be there. But this was the only way to make the prettier VSCode plugin work, see https://github.com/gitKrystan/prettier-plugin-ember-template-tag/issues/38

Reply:
making editors work is pretty important, I'm in favor

Reply:
I am adding a comment there to point to that issue, so we can revert that should that get fixed.